app/models.py

```diff
@@ -22,3 +22,12 @@ class Message(db.Model):
     send_time_unix = db.Column(db.Text())
     subject = db.Column(db.Text())
     thread_id = db.Column(db.Text())
+
+
+class Media(db.Model):
+    __tablename__ = "media"
+
+    id = db.Column(db.Integer, primary_key=True)
+    media_type = db.Column(db.Text())
+    sender_id = db.Column(db.Integer, db.ForeignKey('users.id'))
+    url = db.Column(db.Text())
```

Comment:
Mmm, I'm not sure what the right model should be... but I don't think we necessarily need to have a 1-to-1 mapping between the shape of the API result and the database model.

Maybe the thing that makes the most sense is to do things by message?  We could have an additional column that extracts a url if there's a link there?